activerecord/lib/active_record/connection_adapters/mysql/quoting.rb

```diff
@@ -27,11 +30,11 @@ def cast_bound_value(value)
         end
 
         def quote_column_name(name)
-          self.class.quoted_column_names[name] ||= "`#{super.gsub('`', '``')}`"
+          QUOTED_COLUMN_NAMES[name] ||= "`#{super.gsub('`', '``')}`"
```

Comment:
__question:__ Is there a reason the `quote_table_name` freezes the string prior to memoization, but this does not?

I noticed the `SQLite3::Quoting#quote_column_name` below also omits the `#freeze` call too, so I wondered if maybe there was a reason for it.

Reply:
No reason, wouldn't hurt to freeze it to avoid accidental mutation.